src/solCompile.ts

```diff
@@ -96,14 +101,18 @@ function cliCompile(
       result: JSON.parse(
         execSync(`${solcCommand} --standard-json --allow-paths ${allowPath}`, {
```

Comment:
Change looks good to me.
Can we change the structure so that we don't have the repetition in the two branches

Reply:
done